### DISCUSSION TITLE
When I upload data, I get an error

### DISCUSSION THREAD

**Discussion by kostat-gi** [open-sdg/open-sdg#2092] [Q&A]
hello.

If you change any data in the data store, you will see an error as shown below.
![2024-04-26 164558](https://github.com/open-sdg/open-sdg/assets/103187784/1a0ca9d4-b586-435d-8e87-56f8d86adc4b)

Our repository link is below:
https://github.com/kostat-gi/data

**Comment by sarah-beasley** [comment]
Hi,
Having looked at the error and your data, it looks to be a simple fix! Your data is in csv format, but some indicator data columns are being separated by tabs rather than commas. To fix, change any of the indicators that have tab separated values, to have comma separated values. e.g. 6.2.1 and 9.3.3. This way the columns can be identified correctly. Hope this helps!
